samples/built-in-policy/deploy-logAnalytics-diagnostic-setting-for-datalakeanalytics/azurepolicy.json

```diff
@@ -0,0 +1,129 @@
+{
+  "properties": {
+    "displayName": "Deploy auditing of diagnostic logs in Data Lake Analytics (send to Log Analytics workspace)",
+    "policyType": "BuiltIn",
```

Comment:
Remove this:
    "policyType": "BuiltIn"

Reply:
You'll add this (and the "id" and "name") when you check it into the internal repo as a built-in policy. As a sample, it needs to be possible for any customer to add it as a custom policy in their environment, so it can't have restricted elements (like policyType": "BuiltIn") which are only allowed in built-in policies.

---
In reply to: [288688295](https://github.com/Azure/azure-policy/pull/402#discussion_r288688295) [](ancestors = 288688295)